src/BootstrapTooltipContext.js

```diff
@@ -37,23 +49,27 @@ export default declare("BootstrapTooltip.widget.BootstrapTooltipContext", [_boot
         if (this._contextObj) {
             this.subscribe({
                 guid: this._contextObj.getGuid(),
-                callback: () => this.setTooltipTextAttribute()
+                callback: lang.hitch(this, () => {
```

Comment:
Like below, the semi-colon is missing and this could be an arrow function without body brackets.

Reply:
Done